src/main/resources/i18n/Messages_en.properties

```diff
@@ -116,6 +116,11 @@ manifestCsvTopLevelNameNotUnique=Top Level Variable Name must be unique
 manifestCsvVariableNameNotUniqueWithinParent=No two children of the same parent can have the same name
 manifestCsvVariableParentDoesNotExist=Parent for variable does not exist
 manifestCsvWrongDataTypeForChild=Only tables and sections may be listed as parents
+#{0} is plot name
```

Comment:
FYI, this comment would only show up for `monitoringPlotsNortheastCorner` for translators, not for the other three strings. Phrase treats each comment in a properties file as the description of the string right after the comment.